deps/verifier/src/snp/mod.rs

```diff
@@ -37,6 +37,10 @@ const LOADER_SPL_OID: Oid<'static> = oid!(1.3.6 .1 .4 .1 .3704 .1 .3 .1);
 const KDS_CERT_SITE: &str = "https://kdsintf.amd.com";
 const KDS_VCEK: &str = "/vcek/v1";
 
+// Attestation report versions supported 
```

Comment:
The note here looks very good to be an inline document

```suggestion
/// Attestation report versions supported 
```

Reply:
Fixed. Thanks Ding!